relay-dynamic-config/src/defaults.rs

```diff
@@ -727,6 +727,38 @@ fn span_metrics(
                     .always(),
             ],
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "d:spans/mobile.frames_delay@second".into(),
```

Comment:
Do gauges work, or do we need distributions?

Reply:
Gauges work, I forgot to make that change here. Thanks!